docs/build-deploy.md

```
@@ -0,0 +1,81 @@
+
+| Prefix | Version Bump | Example |
+|--------|-------------|---------|
+| `feat:` | Minor (2.0→2.1) | New game mode |
+| `fix:` | Patch (2.1→2.1.1) | Collision bug fix |
+| `BREAKING CHANGE:` | Major (2.1→3.0) | API change |
+
+**Config files:** `.release-please-config.json`, `.release-please-manifest.json`
+
+## Deployment Targets
+
+### Web
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)